content/en/users/Rucio/Rucio-commands/_index.md

```diff
@@ -0,0 +1,182 @@
+---
+title: "Rucio Commands"
+type: docs
+linkTitle: "Rucio Commands"
+weight: 20
+description: >-
+     Commands to get started with Rucio.
+---
+
+## Commands to get started with Rucio
+
+There are many commands found within Rucio CLI that you may want to become familar with.
+In this guide I will provide a few of the common commands wanted by new users.  
+
+To find more of the commands that you may want to use type `rucio`
+into the containerised client will provide all of the arguments for rucio.
+Typing in the command followed by `-h`,
+or `--help` will provide you with all the options that are available
+as well as some explaination for each.
+
+### ping
+
+Is the simplest command that a user can use to ask the Rucio server which version it is using.
+
+`$ rucio ping`
+
+This checks that there is a connection between the containerised client and the server.
+
+## whoami
+
+Another simple command, which asks the server for the information Rucio has on the current user.
+
+`$ rucio whoami`
+
+This will return output like the following:  
+
+```shell
+status     : ACTIVE
+account    : user1
+account_type : USER
+created_at : YYYY-MM-DDTHH:MM:SS
+updated_at : YYYY-MM-DDTHH:MM:SS
+suspended_at : None
+deleted_at : None
+email      : myemail@domail.country
+```
+
+This ensures that you know which user you are interacting with Rucio as,
+this is very important if you get multiple accounts. But also verifies that the client is set up
+correctly.
+
+### upload
+
+A Rucio command that allows you to upload files from your current environment to any RSE within your VO.
+
+```shell
+$ rucio upload [-h] --rse RSE [--lifetime LIFETIME] [--scope SCOPE]
+                    [--impl IMPL] [--register-after-upload] [--summary]
+                    [--guid GUID] [--protocol PROTOCOL] [--pfn PFN]
+                    [--name NAME] [--transfer-timeout TRANSFER_TIMEOUT]
+                    [--recursive]
+                    args [args ...]
+```
+
+Several of the options you will not need to be used as they will be set by the Rucio Admins
+of your VO when they set up the RSEs.
+Below are a list of options that you may find useful:  
+
+* `RSE` is the Rucio Storage Element or site that you wish to store the data at,
+the list of available RSEs can be seen for your VO with the command
+`rucio list-rses`.
+* `Lifetime` is how long you wish the file to exist,
+not specifyting will make the file permenant until rucio is told to delete it.
+* `Scope` can be used in many ways,
+but often can be an experiment name, or a user space, all users have their own scope assigned to them `user.<username>`.
+* `Register-after-upload` allows for files to be uploaded to the destination,
+and then registered with Rucio, rather than the other way around.
+This can be useful if your connection is intermittent.
+* `Name` is the name of the file that it will be registered to Rucio with,
+if this is not set it will be the name of the file/s provided.
```

Comment:
Typo? (`file/s`)

Reply:
not a typo, but with suggestion of using RSE(s), I have gone with that mostly